lms/djangoapps/discussion/views.py

```diff
@@ -7,23 +7,20 @@
 import logging
 
 from django.contrib.auth.decorators import login_required
-from django.conf import settings
 from django.core.context_processors import csrf
 from django.core.urlresolvers import reverse
 from django.contrib.auth.models import User
 from django.http import Http404, HttpResponseBadRequest
-from django.utils.translation import ugettext_noop
+from django.shortcuts import render_to_response
```

Comment:
Good question. The old one was an edX-specific function that is unable to render templates from inside Django apps (they could only be found in the central templates directory). I borrowed this change from the Teams UI which made the same switch.